lib/ex_aws/ses.ex

```diff
@@ -279,9 +279,9 @@ defmodule ExAws.SES do
     |> flatten_attrs(root)
   end
 
-  defp format_default_template_data(nil), do: "{}"
+  defp format_template_data(nil), do: "{}"
 
-  defp format_default_template_data(default_template_data), do: default_template_data
+  defp format_template_data(template_data), do: template_data |> Poison.encode!()
```

Comment:
For single value piping we usually do:

```suggestion
  defp format_template_data(template_data), do: Poison.encode!(template_data)
```

There is a credo rule for this, maybe we could take the time to add it to our fork?

Reply:
I wont enforce the rule as I want to fork it back to origin, but I'll apply the fixes on my PR